internal/agent/event/zz_from_package.go

```diff
@@ -0,0 +1,16 @@
+package event
```

Comment:
This filename suggests that it is generated.

Reply:
There's a very clear file comment explaining otherwise. Assuming you mean the `zz`, I'm not against removing it.